pkgs/tools/backup/gickup/default.nix

```
@@ -0,0 +1,25 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "gickup";
+  version = "0.10.17";
+
+  src = fetchFromGitHub {
+    owner = "cooperspencer";
+    repo = "gickup";
+    rev = "v${version}";
+    sha256 = "sha256-tiQmb7bBWb99k23lS+d+YR14y4YeYPWqccl/2DLv7Dk=";
+  };
+
+  vendorSha256 = "sha256-DWGrs/ZKMKgVfwU7W+dktLELbW9Co7cmDy9pWVP5p2w=";
+
+  meta = with lib; {
+    description = "Backup all your repositories with Ease.";
```

```suggestion
    description = "Tool to backup repositories";
```

No period at the end (was mentioned in the past in CONTRIBUTING.md), the rest is opinionated.